### DISCUSSION TITLE
How to use Disp8?

### DISCUSSION THREAD

**Discussion by jaredatdannyronsrescue** [DigitalLogicSimCommunity/Digital-Logic-Sim-CE#21] [Q&A] [answered]
Maybe all you electrical engineers out there already know this, but have no idea how to use the Disp8 component.  Any schematics? or a point in the right direction?

**Comment by KrystilizeNevaDies** [comment, marked as answer]
You would set the last 6 bits (first three being ypos, other three being xpos) then set the rgb bits you would like to use for that pixel.

**Comment by jaredatdannyronsrescue** [comment]
Great thanks. This would be good to add to the documentation

**Comment by AOx0** [comment]
Sorry for the bad experience. I changed a little bit `Disp8` to make it clearer for v0.38:

<img width="443" alt="Screen Shot 2022-07-22 at 11 18 50 a m" src="https://user-images.githubusercontent.com/50227494/180481596-5d449f41-6f8c-433c-b3c9-4dd6c740da86.png">
